examples/README.md

```diff
@@ -4,6 +4,10 @@
   - Learn how to write an HTTP service using annotations.
   - See [Annotated services](https://line.github.io/armeria/server-annotated-service.html).
 
+- `grpc-service`
```

Comment:
How about just `grpc`?

Reply:
It caused a build failure because this module depends on `:grpc` module. Gradle thought it's a cyclic dependency. :-(